swagger/oauth2-redirect.html.template.mustache

```diff
@@ -66,4 +66,14 @@
         }
         window.close();
     }
+    // The CSP blocks JS event handlers from inline HTML markup, so instead of
+    // calling run() from the body tag's onload property, as a workaround it
+    // is added here with an event listener.
+    // References:
+    // https://github.com/filamentgroup/loadCSS/issues/312
+    // https://dropbox.tech/security/unsafe-inline-and-nonce-deployment
+    // https://stackoverflow.com/questions/65012573/what-is-the-right-procedure-to-set-a-nonce-in-the-csp-policy#answer-65014515
+    window.addEventListener("load", function() {
```

Comment:
Upstream uses a different event:

https://github.com/swagger-api/swagger-ui/blob/c20a8c479eaa7897a2ddabcaf5f75e6d41f4525c/dev-helpers/oauth2-redirect.html#L68

We should stick with that.